### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by Pandentia** [Thessia/Liara#2]

**Comment by HexadecimalPython** [comment]
We're too lazy to work on that.

**Comment by AraHaan** [comment]
I say doing something like I do documenting the commands per cog file and having people append whatever commands they add to it (weather or not they are added in an cog file). so this commands md file for more information and yes it is an WIP for me as well as I am sadly the only one doing it and documenting stuff alone sucks. [DecoraterBot-devs/DecoraterBot-cogs/Commands.md](https://github.com/DecoraterBot-devs/DecoraterBot-cogs/blob/master/Commands.md)

**Comment by Pandentia** [comment]
No offense, but I think docstrings do a good job on their own already, so making a markdown file listing every command for every cog would just be redundant.

**Comment by AraHaan** [comment]
I do it for convenience for people who do not know how to read python nor docstrings. Also I make my docstrings some odd way just to shut pycharm up. And so I do not use docstrings to explain how to use commands / what it does. It just only ways it is a command for my bot.

**Comment by Risen-phoenix** [comment]
If they can't read docstrings or Python, I don't think this is the right project for them.

**Comment by Pandentia** [comment]
@Risen-phoenix I don't think that that's necessarily the case, however the discord.py help formatter does a good job of displaying the docstrings in an organized fashion, negating the usefulness of a command list.
